templates/base.lds

```diff
@@ -87,24 +87,10 @@ SECTIONS
      * certain core features */
     PROVIDE(__metal_chicken_bit = {{ chicken_bit|default(0) }});
 
-{% if eccscrub_en %}
     /* The memory_ecc_scrub bit is used by _entry code to enable/disable
      * memories scrubbing to zero  */
     PROVIDE(__metal_eccscrub_bit = {{ eccscrub_bit|default(0) }});
```

Comment:
My suggestion to make `__metal_eccscrub_bit` a weak symbol, which you implemented [here](https://github.com/sifive/freedom-metal/commit/81c12a758adfacb5cbebcdc872aa1b7b7ef80542), was motivated in part by saving an entry in the ELF symbol table (when scrubbing is not desired). I think this `PROVIDE` should be predicated on whether or not `eccscrub_bit` is set.